.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+  // jslint extension settings
+  "jslint.version": "2014-07-08",
+  "jslint.options": {
```

Comment:
Please discontinue use of `JSLint`, especially 2014 edition 😃 .

My preference would be to take advantage of TSLint JavaScript linting capabilities instead (that way we need one less extrension in VSCode, and linting should be more consistent between TypeScript and JavaScript files).